02.calendar/calendar.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+
+const main = () => {
+  const today = new Date();
+  const argv = minimist(process.argv.slice(2));
+  const year = argv["y"] || today.getFullYear();
+  const month = argv["m"] || today.getMonth() + 1;
+  const firstDay = new Date(year, month - 1, 1);
+  const lastDay = new Date(year, month, 0);
+  displayHeader(year, month);
+  displayBlankSpaces(firstDay);
+  displayDays(firstDay, lastDay);
+  console.log("\n");
+};
+
+const displayHeader = (year, month) => {
+  const header = `      ${month}月 ${year}\n日 月 火 水 木 金 土`;
+  console.log(header);
+};
+
+const displayBlankSpaces = (firstDay) => {
+  const blankSpacesNumber = firstDay.getDay();
+  for (let number = 1; number <= blankSpacesNumber; number++) {
```

Comment:
```suggestion
  for (let n = 1; n <= blankSpacesNumber; n++) {
```

ループカウンタはnとかiにすることが多いですね。（何か特殊な要件がない限り）

Reply:
修正します!